src/redis/rdb.h

```diff
@@ -103,6 +103,9 @@
 /* Test if a type is an object type. */
 #define rdbIsObjectType(t) ((t >= 0 && t <= 7) || (t >= 9 && t <= 18))
 
+// DFLY EXTENSIONS: WHAT FILE TO PUT THEM?
+#define RDB_OPCODE_FULLSYNC_END 233
+
```

Comment:
maybe add some utility file with defines?

Reply:
I see you already did. I would still write here which range is reserved by Dragonfly and in what file.